dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTablePresenter.java

```diff
@@ -266,38 +265,38 @@ private void initTreeTableView() {
         progress.progressProperty().bind(getServiceTask.progressProperty());
 
         Platform.runLater(() ->
-        getServiceTask.setOnSucceeded(event -> {
-            ds3TreeTable.setPlaceholder(oldPlaceHolder);
-
-            final ObservableList<TreeItem<Ds3TreeTableValue>> children = ds3TreeTable.getRoot().getChildren();
-
-            children.forEach(i -> i.expandedProperty().addListener((observable, oldValue, newValue) -> {
-                final BooleanProperty bb = (BooleanProperty) observable;
-                final TreeItem<Ds3TreeTableValue> bean = (TreeItem<Ds3TreeTableValue>) bb.getBean();
-                ((Ds3TreeTableItem) bean).setDs3TreeTable(ds3TreeTable);
-                ds3Common.getExpandedNodesInfo().put(session.getSessionName() + StringConstants.SESSION_SEPARATOR + session.getEndpoint(), bean);
-            }));
-
-            fileName.setCellFactory(c -> new TreeTableCell<Ds3TreeTableValue, String>() {
-
-                @Override
-                protected void updateItem(final String item, final boolean empty) {
-                    super.updateItem(item, empty);
-                    if (!empty && item.equals(resourceBundle.getString("addMoreButton"))) {
-                        this.getStyleClass().add("styleBold");
-                    } else if (!empty) {
-                        this.getStyleClass().add("styleNormal");
-                    }
-                    this.getStylesheets().add(getClass().getResource("ds3treetable.css").toExternalForm());
-                    setText(item);
-                }
-            });
+                getServiceTask.setOnSucceeded(event -> {
```

Comment:
I'd like to see this in it's own method, and use a method reference, or call the method from within the lambda if possible.  It should help clean up the code.